lib/kubeclient/inflections.rb

```diff
@@ -0,0 +1,4 @@
+ActiveSupport::Inflector.inflections do |inflect|
+  inflect.irregular 'resource_quota', 'resource_quotas'
+  inflect.irregular 'resourcequota', 'resourcequotas'
+end
```

Comment:
@matthewd 
>> I think you can just do inflect.irregular 'quota', 'quotas', and it'll handle it as a suffix.
That works. thanks

@abonas it seems that this is the preferred solution (Inflector)?